charts/sscs-tribunals-api/values.preview.template.yaml

```diff
@@ -538,7 +538,7 @@ elastic:
 sscs-tribunals-frontend:
   enabled: true
   nodejs:
-    image: hmctspublic.azurecr.io/sscs/tribunals-frontend:latest
+    image: hmctspublic.azurecr.io/sscs/tribunals-frontend:pr-1563
```

Comment:
reverty

Reply:
left in for QA, will remember to revert on merge